share/functions.js

```diff
@@ -491,8 +496,9 @@ var ksBuiltin = {
                 if (!command.kill.ring.length)
                     return;
 
-                let (ct = command.getClipboardText())
+                let (ct = command.getClipboardText()) {
```

Comment:
For completeness: actually only this instance gave me problems, but changed the others too as they looked suspect